streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -203,13 +203,14 @@ private void addNewTask(final Task task) {
     }
 
     /**
+     * Tries to initialize any new or still-uninitialized tasks, then checks if they can/have completed restoration.
+     *
      * @throws IllegalStateException If store gets registered after initialized is already finished
      * @throws StreamsException if the store's change log does not contain the partition
      */
-    boolean checkForCompletedRestoration() {
+    boolean tryToCompleteRestoration() {
```

Comment:
Also not necessary, but I feel it better suggests/reminds that it is taking some action here and not just checking a condition